packages/core-bridge/index.d.ts

```diff
@@ -84,6 +84,9 @@ export interface TelemetryOptions {
    * Which determines what tracing data is collected in the Core SDK
    */
   tracingFilter?: string;
+  /** What level, if any, logs should be forwarded from core at */
+  // These strings should match the log::LevelFilter enum in rust
+  logForwardingLevel: 'OFF' | 'ERROR' | 'WARN' | 'INFO' | 'DEBUG' | 'TRACE';
```

Comment:
I'd use this but we have `WARNING` as opposed to `WARN` in node, maybe worth unifying that too.
```suggestion
  logForwardingLevel: 'OFF' | LogLevel;
```